### PR TITLE
Add import export tests

### PR DESCRIPTION
**Pull request by majecty**

### CODE REVIEW

File: remote-trait-object/src/tests/complex_trait.rs [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    }
+}
+
+fn init_logger() {
+    let _ = env_logger::builder().is_test(true).try_init();
+}
+
+fn create_remote_a(port: Arc<dyn Port>) -> ARemote {
+    let a: Arc<dyn A> = Arc::new(SimpleA::new());
+    let handle = port.register(Arc::new(ADispatcher {
```
Comment (junha1) [inline remote-trait-object/src/tests/complex_trait.rs:110]: I think this code is just what the `ImportService` does. Can we just replace with it? Then you will return by `Arc<dyn A>` as well.

File: remote-trait-object/src/service/serde_support.rs [hunk 2]
```diff
@@ -89,3 +89,145 @@ impl<'de, T: ?Sized + Service + ImportService<T>> Deserialize<'de> for SArc<T> {
...
+        }
+        impl FooImpl {
+            pub fn new(handle: u32) -> Self {
+                Self {
+                    handle_to_exchange: HandleToExchange(handle),
+                }
+            }
+        }
+        impl Foo for FooImpl {
+            fn get_handle_to_dispatpch(&self) -> HandleToExchange {
```
[the remark this answers is not in the document]
Comment (majecty) [reply remote-trait-object/src/service/serde_support.rs]: Thanks, I fixed it.

File: remote-trait-object/src/service/serde_support.rs [hunk 3]
```diff
@@ -89,3 +89,145 @@ impl<'de, T: ?Sized + Service + ImportService<T>> Deserialize<'de> for SArc<T> {
...
+            }
+
+            fn register(&self, _service_object: Arc<dyn Dispatch>) -> HandleToExchange {
+                unimplemented!();
+            }
+        }
+
+        /// To make the port not to be dropped, the caller should keep the returned port.
+        pub fn set_global_port() -> Arc<dyn Port> {
+            let port: Arc<dyn Port> = Arc::new(PortMock::new());
```
[the remark this answers is not in the document]
Comment (majecty) [reply remote-trait-object/src/service/serde_support.rs]: Thanks, I fixed it.

File: remote-trait-object/src/tests/complex_trait.rs (hunk 1 above)
Comment (majecty) [reply remote-trait-object/src/tests/complex_trait.rs]: Thanks, I fixed it.

**Pull request merged**